street_level_crime/test.py

```diff
@@ -69,3 +71,11 @@ def test_process_line_skip_none(self):
 
 if __name__ == '__main__':
     unittest.main()
+
+
+class TestConvertToCSV(unittest.TestCase):
+    def test_convert_to_csv(self):
+        headers = ['a', 'b', 'c']
+        data = [['a', 'b', 'c'], ['d', 'e', 'f'], ['g', 'h', 'i'], ['j', 'k', 'l'], ['m', 'n', 'o']]
+        with mock.patch('builtins.input', return_value="Jack"):
```

Comment:
Testing file writing is a tricky one, would likely need to be a manual check

Reply:
Removed for now